browser-tests/playwright.config.ts

```diff
@@ -1,5 +1,7 @@
-const { devices } = require("@playwright/test");
-require("dotenv").config();
+import { devices } from "@playwright/test";
+import dotenv from "dotenv";
+
+dotenv.config();
 
 module.exports = {
```

Comment:
Is there a reason why we still use `module.exports` here instead of an es6 export? Could something like this work?
```
export default defineConfig({
  workers: 6,
  projects: [
    {
      name: "chromium",
      use: {
        ...devices["Desktop Chrome"],
        headless: true,
      },
    },
  ],
  use: {
    baseURL: process.env?.WEBSITE_HOST || "http://localhost:4601",
    headless: true,
  },
})
```

Reply:
And just one tiny thing, I think line 21 in the browser tests README needs to be updated to `playwright.config.ts`